frontend/src/chooseAdventure/introView.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import * as PropTypes from "prop-types";
+
+
+/**
+ * Component for displaying choose your own adventure skeleton
+ */
+
+class IntroView extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <div>
+                <div className='row'>
+                    <div className={'col-6'}>
+                        {this.props.desc}
+                    </div>
+                    <img className= 'col-6' src='/static/img/sample.jpg' alt="Sample" />
+
+                </div>
+
+                <button className='col-5 cyoa-button' onClick={() => this.props.setView('stage')}>
+                    Get started
+                </button>
+            </div>
+
```

Comment:
i know this is just mvp / prototyping code but whoa there unnecessary whitespace